### PR TITLE
HHH-17243 - java.time.Duration handling in Envers

### PR DESCRIPTION
**Pull request by KaiSuchomel**
https://hibernate.atlassian.net/browse/HHH-17243

### CODE REVIEW

File: hibernate-core/src/main/java/org/hibernate/boot/model/process/spi/MetadataBuildingProcess.java [hunk 1]
```diff
@@ -672,7 +672,7 @@ public void contributeAttributeConverter(Class<? extends AttributeConverter<?, ?
 					jdbcTypeRegistry,
 					preferredSqlTypeCodeForDuration,
 					Duration.class,
-					"duration"
+					Duration.class.getSimpleName(), Duration.class.getName()
```
Comment (beikov) [inline hibernate-core/src/main/java/org/hibernate/boot/model/process/spi/MetadataBuildingProcess.java:675]:
```suggestion
					StandardBasicTypes.DURATION.getName()
```

**Comment by beikov** [comment]
Thanks!

**Comment by KaiSuchomel** [comment]
@beikov Can you ping me if you know in which quarkus the fix will be available??

**Comment by yrodiere** [comment]
Hey @KaiSuchomel ,

I'm part of the Hibernate team and we're trying to relicense Hibernate Envers to ASL2. I would love to have your explicit approval to re-license your contributions. Can you please give it here, or over email at `yoann <at> hibernate.org`?

Thanks.

---

By the way, regarding your question, I assume you found out, but sorry for the lack of answer :) The Hibernate team generally doesn't know this in advance, it depends when Hibernate ORM gets released and whether it makes it into the next release window of Quarkus.

**Comment by KaiSuchomel** [comment]
@yrodiere You have my approval 👍

**Comment by yrodiere** [comment]
Thank you!

**Pull request merged**